src/assets/scripts/groepen/Civikaartje.jsx

```diff
@@ -0,0 +1,40 @@
+var React = require("react");
+var $ = require("jquery");
+var _ = require("underscore");
+
+class Civikaartje extends React.Component {
+
+  constructor(props, context) {
+    super(props);
+    this.profiel = props.profiel;
+    this.context = context;
```

Comment:
Unless I'm missing something context does NOT need to be passed around explicitly. I think you can remove both. If something breaks, pass it to super.